compiler/luci-interpreter/pal/cmsisnn/pal.cmake

```
@@ -63,3 +63,5 @@ macro(add_pal_to_target TGT)
 
     target_link_libraries(${TGT} PRIVATE luci_interpreter_cmsisnn_pal)
 endmacro()
+
+message(FATAL_ERROR "pal/mcu is disabled. please use onert-micro")
```

Maybe?

```suggestion
message(FATAL_ERROR "pal/cmsisnn is disabled. please use onert-micro")
```

This seems not important. It would be better to remove cmsisnn and mcu directories to avoid duplication with onert-micro.